clang/test/Driver/sycl-lto.cpp

```diff
@@ -8,7 +8,12 @@
 // RUN: not %clangxx -fsycl --offload-new-driver -foffload-lto=thin -fsycl-device-code-split=off %s -### 2>&1 | FileCheck -check-prefix=CHECK_SPLIT_ERROR %s
 // CHECK_SPLIT_ERROR: '-fsycl-device-code-split=off' is not supported when '-foffload-lto=thin' is set with '-fsycl'
 
-// Verify there's no error and we see the expected cc1 flags with the new offload driver.
+// Verify there's no error and we see the expected cc1 flags and tool invocations with the new offload driver.
 // RUN: %clangxx -fsycl --offload-new-driver -foffload-lto=thin %s -### 2>&1 | FileCheck -check-prefix=CHECK_SUPPORTED %s
 // CHECK_SUPPORTED: clang{{.*}} "-cc1" "-triple" "spir64-unknown-unknown" {{.*}} "-flto=thin" "-flto-unit"
+// CHECK_SUPPORTED: sycl-post-link{{.*}}
+// CHECK_SUPPORTED-NEXT: file-table-tform{{.*}}
+// CHECK_SUPPORTED-NEXT: llvm-foreach{{.*}} "--" {{.*}}clang{{.*}} "-fsycl-is-device"{{.*}} "-flto=thin" "-flto-unit"
```

Comment:
why do we need to call clang here?

Reply:
if there were no `sycl-post-link`, we would be calling clang with these args (to add thinLTO info and probably do some stuff I don't know about), this just makes sure we do the same thing on each output of `sycl-post-link`